packages/contracts-bedrock/contracts/universal/CrossDomainMessenger.sol

```diff
@@ -345,17 +361,36 @@ abstract contract CrossDomainMessenger is
             "CrossDomainMessenger: message has already been relayed"
         );
 
+        // If there is not enough gas left to perform the external call and finish the execution,
+        // return early and assign the message to the failedMessages mapping.
+        // We are asserting that we have enough gas to:
+        // 1. Call the target contract (_minGasLimit + RELAY_CALL_OVERHEAD + RELAY_GAS_CHECK_BUFFER)
```

Comment:
Should this be updated to:

```suggestion
        // 1. Call the target contract (_minGasLimit + RELAY_CALL_OVERHEAD + RELAY_RESERVED_GAS + RELAY_GAS_CHECK_BUFFER)
```

Reply:
The `RELAY_RESERVED_GAS` is reserved for the execution after the call - see #2 😄